CONTRIBUTING.md

```diff
@@ -0,0 +1,94 @@
+# Contributing
+
+## Getting Started
+
+General information about setting up your Python environment, testing modules,
+Ansible coding styles, and more can be found in the [Ansible Community Guide](
+https://docs.ansible.com/ansible/latest/community/index.html).
+
+
+## ServiceNow ITSM Collections
+
+### servicenow.itsm
+
+The Ansible Collection for ServiceNow IT Service Management ([ITSM](https://www.servicenow.com/products/itsm.html)) includes a variety of Ansible content to help automate the management of ServiceNow IT Service Management.
+
+New modules and plugins developed by the community should be proposed to `servicenow.itsm`.
+
+## Submitting Issues
+All software has bugs, and the `servicenow.itsm` collection is no exception. When you find a bug,
+you can help tremendously by [telling us about it](https://github.com/ansible-collections/servicenow.itsm/issues).
+
+If you should discover that the bug you're trying to file already exists in an issue,
+you can help by verifying the behavior of the reported bug with a comment in that
+issue, or by reporting any additional information.
+
+## Environment setup
+1. Sign up for a [ServiceNow ID](https://signon.service-now.com/x_snc_sso_auth.do#/sign-up) 
+1. Request and setup an Instance. 
+1. When the instance is ready, you will be redirected to the Developer Portal `e.g. https://dev12345.service-now.com/`. In All, search for Application Registry then New - Create an OAuth API endpoint for external clients.
+1. Ensure that you have changed the Integration tests configuration file accordingly with your Personal Developer Instance credentials for connection and access. A template is available [here](https://github.com/ansible-collections/servicenow.itsm/blob/main/tests/integration/integration_config.yml.template).
+1. For more information, check [General usage pattenrs](https://github.com/ansible-collections/servicenow.itsm/blob/main/docs/general_usage_patterns.md).
+
```

Comment:
I think it is necessary to mention here that docker or podman is installed locally as all tests (sanity/units/integration) are executed in docker when ran through 'make'.

Reply:
That sounds reasonable.